src/HelperLocator.php

```diff
@@ -88,6 +47,16 @@ public function has(string $name) : bool
 
         if (function_exists($name)) {
             $this->instances[$name] = $func;
+
+            return true;
+        }
+
+        $mabeyHelper = 'Qiq\Helper\\' . $name;
+        if (class_exists($mabeyHelper)) {
```

Comment:
So here the constaint is your class namespace should be `Qiq\Helper` . I am not sure if that has some advantage when the project namespace is different.

Reply:
I think @koriym means for this as a "final fallback" to find the bundled Qiq helpers, so that you don't have to register them with the locator. @koriym am I right about that?

Reply:
@pmjones Yes, you can put the helper in a specific plugin folder to make it available without registering it with locater. This is similar to Smarty's plugin.

By making it an independent file, it is easier to test, move, and share.

@harikt I figured that qiq namespace is better than application namespace for sharing.

Reply:
After users create some helper plugins, they may want to reuse them in other projects or publish them on packagist in public. This is difficult with the current register method.

Reply:
I wrote my comment on https://github.com/qiqphp/qiq/pull/7/files#r802259779 . Because I want to hint there.